go/tasks/pluginmachinery/flytek8s/pod_helper.go

```diff
@@ -202,7 +216,10 @@ func ConvertPodFailureToError(status v1.PodStatus) (string, string) {
 		if containerState.Terminated != nil {
 			if strings.Contains(c.State.Terminated.Reason, OOMKilled) {
```

Comment:
lets add some documentation here, why we are doing this

Reply:
sure